douane.c

```diff
@@ -1221,7 +1226,7 @@ static void __exit exit_module(void)
     activities_socket = NULL;
   }
 
-  nf_unregister_hook(&nfho_outgoing);
+  nf_unregister_net_hook(&init_net, &nfho_outgoing);
```

Comment:
The compilation condition you did for the `nf_register_net_hook` is required here too. Please add it.

Reply:
Yes of course ... it was late already when i implemented this 😄